modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -158,6 +158,10 @@ spec:
               value: <target_namespace>
             - name: CONFIGMAP_NAME
               value: kubevirt-vm-latency-checkup-config
+            - name: POD_UID
```

Comment:
This environment variable value is populated with the Pod's UID from the downward API:
https://kubernetes.io/docs/concepts/workloads/pods/downward-api/

This value is used to by the checkup to mark the two VMIs as owned by the checkup, so they could be garbage-collected.

IMO we should look at it as a boilerplate (same as in the DPDK checkup).

Reply:
I agree about treating this as a boilerplate, but I still think the user should know the meaning of this env. var. (even if they are not supposed to change it), as this is not an intuitive var (at least per my understanding).
What do you think about something like:
An environment variable, used to mark the VMs that are created by the checkup as owned by the job's pod, in order to ensure that the VMs are eventually garbage-collected.

Reply:
I think its a good idea to document the purpose of this environment variable, but I think we should do it on a follow-up PR so it could be done for both VM latency and DPDK checkups.

Reply:
I accept this suggestion.